### PR TITLE
Add Consistent Date Shifting Masking Provider

### PR DESCRIPTION
**Pull request by jarrettranderson**

### CODE REVIEW

File: whc-shared/src/main/java/com/ibm/whc/deid/shared/pojo/config/masking/DateTimeConsistentShiftMaskingProviderConfig.java [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  public static enum DateShiftDirection {
+
+    /**
+     * The shifted value is before the original value.
+     */
+    before,
+
+    /**
+     * The shifted value can be before or after the original value.
+     */
```
Comment (dumashwang) [inline whc-shared/src/main/java/com/ibm/whc/deid/shared/pojo/config/masking/DateTimeConsistentShiftMaskingProviderConfig.java:35]: I think we usually define enum in upper case? If we want the value to be displayed as lower case, we can use annotations(@JsonProperty?)

**Review by dumashwang — APPROVED** [review]
Looks good.

File: whc-shared/src/main/java/com/ibm/whc/deid/shared/pojo/config/masking/DateTimeConsistentShiftMaskingProviderConfig.java (hunk 1 above)
Comment (jarrettranderson) [reply whc-shared/src/main/java/com/ibm/whc/deid/shared/pojo/config/masking/DateTimeConsistentShiftMaskingProviderConfig.java]: changing

**Comment by jarrettranderson** [comment]
Additional change sets approved in fork.

**Pull request merged**